lib/services/platform-service.ts

```diff
@@ -67,6 +67,10 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 		}
 
 		let platformData = this.$platformsData.getPlatformData(platform, projectData);
+		let currentPlatformData: any = this.$projectDataService.getNSValue(projectData.projectDir, platformData.frameworkPackageName);
```

Comment:
`this.$projectDataService.getNSValue` can return `null` can you add check if it is null before you try to access `currentPlatformData[constants.VERSION_STRING]`